Sources/Features/Client/BacktraceReporter.swift

```diff
@@ -21,7 +22,9 @@ final class BacktraceReporter {
                              credentials: credentials,
                              repository: try PersistentRepository<BacktraceReport>(settings: dbSettings))
         self.repository = try PersistentRepository<BacktraceReport>(settings: dbSettings)
-        self.attributesProvider = AttributesProvider()
+        let localAttributeProvider = AttributesProvider()
```

Comment:
What does `let` mean here?

Reply:
Swift compiler also warn me about var usage. I think we should stick with let - instead of var - but I'm also open for other suggestions.